requirements/dev.txt

```diff
@@ -1,13 +1,13 @@
 coverage==4.3.4
-tox==2.2.1
 pytest-cov==2.4.0
 # astroid > 2.0.4 is not compatible with pylint1.7
 astroid>=1.5.8,<2.1.0
 pylint==1.7.2
 
 # Test requirements
-pytest==3.1.0
-py==1.4.33
+pytest==3.6.0
```

Comment:
This might cause some weird test failures/warnings I think.

Reply:
Are you talking about removing py or the pytest upgrade?

Reply:
I had to upgrade pytest to even be able to use pytest-timeouts.